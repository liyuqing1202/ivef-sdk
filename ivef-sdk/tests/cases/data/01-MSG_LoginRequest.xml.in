<MSG_IVEF xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">                 
	<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="1.2"/>
	<Body>
		<LoginRequest Encryption="1" Name="John Doe" Password="Very Secret"/>
	</Body>
</MSG_IVEF>
