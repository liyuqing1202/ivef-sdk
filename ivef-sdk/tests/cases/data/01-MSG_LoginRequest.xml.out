TestApp ready for input
MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 1.2
    Body
        LoginRequest
            Encryption = 1
            Name = John Doe
            Password = Very Secret
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="1.2"/>
<Body>
<LoginRequest Encryption="1" Name="John Doe" Password="Very Secret"/>
</Body>
</MSG_IVEF>

TestApp shutting down
